test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/tasks/main.yml

```diff
@@ -0,0 +1,128 @@
+---
+## Task file for setup/teardown AWS resources for aws_ssm integration testing
+- block:
+    - name: set up aws connection info
+      set_fact:
+        aws_connection_info: &aws_connection_info
+          aws_access_key: "{{aws_access_key}}"
+          aws_secret_key: "{{aws_secret_key}}"
+          security_token: "{{security_token}}"
+          region: "{{aws_region}}"
+      no_log: yes
+
+    - name: Install Session Manager Plugin for Debian/Ubuntu
+      include_tasks: debian.yml
+      when: ansible_distribution == "Ubuntu" or ansible_distribution == "Debian"
+
+    - name: Install Session Manager Plugin for RedHat/Amazon
+      include_tasks: redhat.yml
+      when: ansible_distribution == "CentOS" or ansible_distribution == "RedHat" or ansible_distribution == "Amazon"
+
+    - name: Install Boto3
+      pip:
+        name: boto3
+
+    - name: Install Boto
+      pip:
+        name: boto
+
+    - name: Ensure IAM instance role exists
+      iam_role:
+        name: "{{resource_prefix}}-aws-ssm-role"
```

Comment:
Hi @hanumantharaomvl, ok I think I see the issue.  The policy restricts CreateRole to certain resource matches - https://github.com/mattclay/aws-terminator/blob/master/aws/policy/security-services.yaml#L16:L19  The policy the test is creating here does not match these patterns - I'll have to look into why it's passing that point when I launch the tests from outside shippable.  If you update this to `ansible-test-{{resource_prefix}}-aws-ssm-role` that should match the policy.